packages/earn-controller/src/EarnController.ts

```diff
@@ -40,27 +49,40 @@ import type {
 export const controllerName = 'EarnController';
 
 export type PooledStakingState = {
-  pooledStakes: PooledStake;
-  exchangeRate: string;
-  vaultMetadata: VaultData;
-  vaultDailyApys: VaultDailyApy[];
-  vaultApyAverages: VaultApyAverages;
+  [chainId: number]: {
```

Comment:
Since this type is exported, these changes look like they're breaking to me, what do you think about documenting them in the changelog?